### PR TITLE
[tests-only][full-ci] Add spaces example for apiWebdavUpload1 suite

### PR DESCRIPTION
**Pull request by SagarGi**
## Description
 Added Spaces for suite apiWebdavUpload1 

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/3124

## How Has This Been Tested?

- CI
- Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavUpload1/uploadFile.feature [hunk 1]
```diff
@@ -205,6 +277,11 @@ Feature: upload file
       | old         |
       | new         |
 
+    @skipOnOcV10 @personalSpace
+    Examples:
+      | dav_version |
+      | spaces      |
+
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavUpload1/uploadFile.feature:284]:
```suggestion


```

Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavUpload1/uploadFile.feature:284]: check for this type of spacings below too!

File: tests/acceptance/features/apiWebdavUpload1/uploadFile.feature [hunk 2]
```diff
@@ -165,6 +222,11 @@ Feature: upload file
...
   @issue-ocis-reva-174
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiWebdavUpload1/uploadFile.feature:230]: i think this has been closed.you can removed it. Isn't it @kiranparajuli589  dai?

Comment (phil-davis) [reply tests/acceptance/features/apiWebdavUpload1/uploadFile.feature:230]: yes, while we are touching core API test feature files, we can remove tags that mention  old oCIS and reva issues. But we don't have to - they don't do any harm. "One day" we could do a PR to remove them all.

Note: in this case the tag refers to an issue that was in the https://github.com/owncloud/ocis-reva repo, which was archived a long time ago. So it is a really old issue.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
@kiranparajuli589 @amrita-shrestha please also review and merge if it looks OK to you.

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**